complex/ft_main.c

```diff
@@ -471,6 +472,8 @@ static void ft_fw_usage(char *program)
 	FT_PRINT_OPTS_USAGE("-x", "exit after test run");
 	fprintf(stderr, "\nClient only options:\n");
 	FT_PRINT_OPTS_USAGE("-f <test_config_file>", "");
+	FT_PRINT_OPTS_USAGE("-a <provider_name>", "");
+	FT_PRINT_OPTS_USAGE("-t <test_name>", "");
```

Comment:
In runfabtests.sh, it doesn't use any letter for specifying the provider name. I can use '-u' for config option but if I add -f for sockets in runfabtests.sh that would affect all the people who are using without "-f". Not sure if that'll be a good idea.